### ISSUE TITLE
Download single file to folder

### ISSUE THREAD

**Issue by ghost** [weslly/Nettuts-Fetch#3]
It would be great if the behaviour for fetching a single file was to download that file to a configurable location in a project, rather than to paste the content into the currently open file. Would make this insanely useful.

**Comment by ghost** [comment]
Ctrl + N, Fetch single file, Ctrl + S, Pick Directory

Although, I agree, it'd be nice to simply Fetch and type directory (functionality of Fetch Package).

**Comment by ghost** [comment]
Yeah unfortunately while that is a decent solution to the problem, it's not convenient enough to justify using fetch to me. Too much thinking :P.
Ideally you could set a default location (e.g currentproject/js/libs/) and with one command it pulls down the file and places it in that location.

**Comment by ghost** [comment]
I completely agree and rarely use it. It was such a good idea, but just falls short. A few more hours of code and Fetch would have been awesome. :(

**Comment by bgallagh3r** [comment]
Ya it would be nice to be able to add a relative path in the Fetch settings when you pull a file.

Like if I was pulling jQuery I know i always want it to be in /js/libs/ it would be nice to maybe add an optional "relative_path:" setting to tell it to look for that path first, if it doesn't exist then just open in it a file.

**Comment by zelig** [comment]
it would be very reasonable to have this feature to set a path to files and packages. Also quite counterintuitive that looking at the settings file you do not know where the local target for the remote source.
